### DISCUSSION TITLE
"An unexpected error occured" message appears in all Wikijs linked URLs

### DISCUSSION THREAD

**Discussion by mohamed-rouissi** [requarks/wiki#4591] [Error / Bug Report]
Hi everyone, 

First things first, below are all the system informations:
- _OS_: Ubuntu 20.04.3 LTS (VM)
- _Nodejs_: v14.18.1
- _Wikijs_: 2.5.219
- _Postgresql_: 12.8 

After the installation, comes the registration page where I have to enter the administration information. Once this is done, the first malfunction: there is no automatic redirection to the login page. To access the login page, I have to restart the `wikijs` daemon and manually re-enter the URL. 

Then comes the heart of the problem: in each new URL of the application I access, the error "an expected error occurred" appears:

- **Login Page:**
![error1](https://user-images.githubusercontent.com/61789098/137315280-28e420f4-ff1c-4080-931c-17f3833f83cf.jpg)

- **New Page:**
![error2](https://user-images.githubusercontent.com/61789098/137315725-dd74566b-9d4b-48a1-a28e-1d21de915db3.jpg)

- **Home Page:**
![error3](https://user-images.githubusercontent.com/61789098/137315867-7b7381e8-9da9-45e4-ae70-093abc1d2a26.jpg)

For debugging, I downgraded Wikijs and tried the 2.4.107 version and changed the DB (MySQL), same problem. 

Please note that the application is functional but this error message is everywhere and appears for 5 seconds and then disappears.

Could someone tell me if this is a bug or if my installation/configuration is incorrect? Every information is appreciated. 

Thanks.

**Comment by NGPixel** [comment]
Does your wiki have internet access? Looking at your screenshots, the locale strings fail to load (which would explain the error message you get on every page).

**Comment by mohamed-rouissi** [comment]
Yes sure it has, I downloaded all the repos and the packages through the Internet. I work behind a proxy, and all the configs have been correctly implemented too. I'll take a look at the locale as well.

**Comment by mohamed-rouissi** [comment]
Sorry for the delay. So I verified the locale and everything seems to be okay. Below is the output:

```
root@wiki:/home/admit# locale
LANG=en_US.UTF-8
LANGUAGE=
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=de_DE.UTF-8
LC_TIME=de_DE.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=de_DE.UTF-8
LC_NAME=de_DE.UTF-8
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=de_DE.UTF-8
LC_MEASUREMENT=de_DE.UTF-8
LC_IDENTIFICATION=de_DE.UTF-8
LC_ALL=
```

**Reply by crbuhian** [reply]
Hi... Were you able to fix this issue? I'm running to the same issue as well.. Appreciate any insight you might have. Thanks!

**Comment by sikujua** [comment]
Am facing the exact problem, installation is not on docker, its directly on a linux host
